demo/demo.js

```diff
@@ -1,75 +1,61 @@
 /* eslint-disable jsx-a11y/media-has-caption, class-methods-use-this */
-import React, { PureComponent } from 'react';
+import React, { useState } from 'react';
 import ReactDOM from 'react-dom'; // eslint-disable-line
 import ReactCrop from '../lib/ReactCrop';
```

Comment:
I had to make to following changes to be able to compile the code.
import ReactCrop from 'react-image-crop';
import 'react-image-crop/dist/ReactCrop.css';